<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(212, 212, 212);</string>
  </property>
  <widget class="QLabel" name="labelAnoboy">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/newPrefix/cats-c6511.jpg)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Anoboy.qrc">:/newPrefix/cats-c6511.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_FirstTime">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>461</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(204, 0, 0);</string>
   </property>
   <property name="text">
    <string>      First Time Setup</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMediaP">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tempus Sans ITC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 85, 0);</string>
   </property>
   <property name="text">
    <string>                  1. Preferred Media Player :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTermux">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tempus Sans ITC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 85, 0);</string>
   </property>
   <property name="text">
    <string>                 2. Are you Using Termux ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLogin">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>470</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tempus Sans ITC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 85, 0);</string>
   </property>
   <property name="text">
    <string>                         3. Input Token :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>520</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSubmit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>590</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Anoboy.qrc"/>
 </resources>
 <connections/>
</ui>
